src/epcpm/importexport.py

```diff
@@ -311,6 +314,19 @@ def full_export(
         skip_output=skip_output,
     )
 
+    epcpm.anomaliestoc.export(
+        h_path=paths.anomalies_h,
+        anomaly_model=project.models.anomalies,
+        parameters_model=project.models.parameters,
+    )
+
+    epcpm.anomaliestoxlsx.export(
+        path=paths.anomalies_spreadsheet,
+        anomaly_model=project.models.anomalies,
+        parameters_model=project.models.parameters,
+        skip_output=skip_output,
```

Comment:
Just so you know, `skip_output` is true when the SunSpec output is skipped. This may or may not be the behavior you want?

Reply:
Ok, if it is sunspec-specific flag I won't utilize it in anomaly generation